shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsTest.mm

```diff
@@ -143,4 +144,128 @@ - (void)testCanCreatePlatformViewWithoutFlutterView {
   flutterPlatformViewsController->Reset();
 }
 
+- (void)testChildClippingViewHitTests {
+  ChildClippingView *childCilppingView = [[[ChildClippingView alloc] initWithFrame:CGRectMake(0, 0, 500, 500)] autorelease];
+  UIView *childView = [[[UIView alloc] initWithFrame:CGRectMake(100, 100, 100, 100)] autorelease];
+  [childCilppingView addSubview:childView];
+
+  XCTAssertFalse([childCilppingView pointInside:CGPointMake(50, 50) withEvent:nil]);
+  XCTAssertTrue([childCilppingView pointInside:CGPointMake(150, 150) withEvent:nil]);
```

Comment:
you might want to check the edges as well.  e.g. `100,100`, `99,99`, `100,99`, `99,100`.

Reply:
same with the right, bottom edge

Reply:
done